.github/workflows/builds.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish docs via GitHub Pages
```

Comment:
Remember to begin YAML files with a `---` line. That will be caught by YAML lint tooling.

Reply:
Also, let's prefer `.yaml` over `.yml`.

Reply:
Fixed in 506c9a6.